src/patternfly/components/DataList/data-list.scss

```diff
@@ -238,6 +245,65 @@
     box-shadow: var(--pf-c-data-list__item--m-selected--BoxShadow);
   }
 
+  // stylelint-disable
+  &.pf-m-draggable {
+    position: relative;
+    cursor: pointer;
+
+    .pf-c-data-list__item-draggable-icon {
+      color: var(--pf-c-data-list__item--m-draggable-draggable-icon--Color);
+    }
+
+    &:hover:not(.pf-m-dragged) {
+      z-index: var(--pf-c-data-list__item--m-draggable--hover--ZIndex);
+      box-shadow: var(--pf-c-data-list__item--m-draggable--hover--BoxShadow);
+
+      &:not(:last-child) {
+        --pf-c-data-list__item--BorderBottomWidth: 0;
+
+        + .pf-c-data-list__item {
+          border-top: var(--pf-c-data-list__item--m-selectable--hover--item--BorderTopWidth) solid var(--pf-c-data-list__item--m-selectable--hover--item--BorderTopColor);
+        }
+      }
+
+      .pf-c-data-list__item-draggable-icon {
+        --pf-c-data-list__item--m-draggable-draggable-icon--Color: var(--pf-c-data-list__item--m-draggable--hover-draggable-icon--Color);
+      }
+    }
+
+    &.pf-m-disabled {
+      pointer-events: none;
+
+      .pf-c-data-list__item-draggable-icon {
+        --pf-c-data-list__item--m-draggable-draggable-icon--Color: var(--pf-c-data-list__item--m-draggable--m-disabled-draggable-icon--Color);
+      }
+    }
+
+    &.pf-m-dragged {
```

Comment:
wdyt about `.pf-m-dragging` instead since it's more of a present tense action than past tense?

Reply:
yep thats a good idea